Thu Nov  3 00:43:58 2022 - INFO  - Start to prepare the test environment.
Failed to enable unit: Unit file pmcd.service does not exist.
Failed to start pmcd.service: Unit pmcd.service not found.
Failed to enable unit: Unit file pmlogger.service does not exist.
Failed to start pmlogger.service: Unit pmlogger.service not found.
Thu Nov  3 00:46:01 2022 - INFO  - End to prepare the test environment.
Thu Nov  3 00:46:04 2022 - INFO  - Start to run test.
oe_test_pminfo_03.sh:行30: pminfo：未找到命令
Thu Nov  3 00:46:07 2022 - ERROR - oe_test_pminfo_03.sh line 31
oe_test_pminfo_03.sh:行32: pminfo：未找到命令
Thu Nov  3 00:46:11 2022 - ERROR - oe_test_pminfo_03.sh line 33
oe_test_pminfo_03.sh:行34: pminfo：未找到命令
Thu Nov  3 00:46:14 2022 - ERROR - oe_test_pminfo_03.sh line 35
oe_test_pminfo_03.sh:行36: pminfo：未找到命令
Thu Nov  3 00:46:17 2022 - ERROR - oe_test_pminfo_03.sh line 37
oe_test_pminfo_03.sh:行38: pminfo：未找到命令
Thu Nov  3 00:46:20 2022 - ERROR - oe_test_pminfo_03.sh line 39
oe_test_pminfo_03.sh:行40: pminfo：未找到命令
Thu Nov  3 00:46:24 2022 - ERROR - oe_test_pminfo_03.sh line 41
oe_test_pminfo_03.sh:行42: pminfo：未找到命令
Thu Nov  3 00:46:28 2022 - ERROR - oe_test_pminfo_03.sh line 43
oe_test_pminfo_03.sh:行44: pminfo：未找到命令
Thu Nov  3 00:46:31 2022 - ERROR - oe_test_pminfo_03.sh line 45
oe_test_pminfo_03.sh:行46: pminfo：未找到命令
Thu Nov  3 00:46:33 2022 - ERROR - oe_test_pminfo_03.sh line 47
oe_test_pminfo_03.sh:行48: pminfo：未找到命令
Thu Nov  3 00:46:34 2022 - ERROR - oe_test_pminfo_03.sh line 49
Thu Nov  3 00:46:37 2022 - INFO  - End to run test.
Thu Nov  3 00:46:38 2022 - ERROR - failed to execute the case.
Thu Nov  3 00:46:40 2022 - INFO  - Start to restore the test environment.
Traceback (most recent call last):
  File "/root/mugen/mugen-riscv/libs/locallibs/rpm_manage.py", line 173, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/mugen/mugen-riscv/libs/locallibs/rpm_manage.py", line 144, in rpm_remove
    with open(tmpfile, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'Last metadata expiration check: 0:32:20 ago on 2022年11月03日 星期四 00时13分08秒.\nNo match for argument: pcp\nError: Unable to find a match: pcp'
Thu Nov  3 00:46:52 2022 - INFO  - End to restore the test environment.
